### DISCUSSION TITLE
hello_world.py in https://14946363-23cd-404a-a700-73e1a60ec29c.internal.dedicated.h2o.ai/#hello_world needs to add main to the import

### DISCUSSION THREAD

**Discussion by wendycwong** [h2oai/wave#2168] [Q&A] [answered]
I was trying to learn how to write an app and notice a problem with hello_world.py:

from h2o_wave import site, ui

The problem is with the import, it needs to import main as well.
Quick fix is just to do this:

from h2o_wave import site, ui, main

**Comment by mturoci** [comment, marked as answer]
Hey @wendycwong!

The `main` import is missing because `hello_world` example is a [Wave script](https://wave.h2o.ai/docs/scripts/) and not a [Wave app](https://wave.h2o.ai/docs/apps/).
